package.json

```diff
@@ -3,37 +3,53 @@
   "version": "7.0.0-beta.0",
   "description": "The router for easy microfrontends",
   "type": "module",
-  "main": "lib/umd/single-spa.min.cjs",
+  "main": "./lib/umd/single-spa.min.cjs",
```

Comment:
main/types pair always comes as CJS, since it is used with node10 resolution which only reads those fields, and only supports CJS format.

Reply:
The `"main"` field is used by nodejs and bundlers, not just typescript. Since the type is module, this should be an es module. Let's change it.

Reply:
We can switch it, but this behaviour is unexpected.

https://rollupjs.org/introduction/#publishing-es-modules
> To make sure your ES modules are immediately usable by tools that work with CommonJS such as Node.js and webpack, you can use Rollup to compile to UMD or CommonJS format, and then point to that compiled version with the main property in your package.json file.

https://esbuild.github.io/api/#main-fields-for-package-authors
>The main field is expected to be CommonJS[...]

Reply:
I'd rather see the bug report than continue being harassed by Node10 module resolution within Typescript. I took away your write access because of failure to obey the single-spa governance document at https://single-spa.js.org/governance/. The rollup documentation you linked to says "you can." As for esbuild, they do not determine how package.jsons work better than node or webpack.

You've not been able to accept that I am the decision maker for single-spa several times, after stalking my social media account and DMing me about it. You pushed es-module-shims when I said no. You pushed monorepos when I said no. You prefer speaking on Github rather than Slack since in Slack I send you links of hand-crafted repositories proving what you say to be false.

I will change the main field to an ES module, as the creator of single-spa, per the governance document.